### ISSUE TITLE
Jacobi and Legendre symbol on `openssl::bn::BigNum`

### ISSUE THREAD

**Issue by lowlevl** [clux/quadratic#4]
Hello,

since I'm going to need to implement the Jacobi symbol computation for [`openssl::bn::BigNum`](https://docs.rs/openssl/0.10.36/openssl/bn/struct.BigNum.html), would you be interested in including my implementation of this in your library with a `feature` (named something like `openssl`) when I'm done ?

**Comment by clux** [comment]
Yeah, sure, if you feel that this can be extended in a nice way. Note that this library was a learning project and it hasn't been updated since it was made (even CI provider was dead).

I would probably prefer doing an implementation like this generically with something like [num-bigint](https://crates.io/crates/num-bigint) and then have the openssl specific conversion from that type.
